tools/check-links

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
```

Comment:
Include copyright comments at the top and bottom of this file. See other Python code for examples.